### ISSUE TITLE
Header with icon has unnecessary padding-left

### ISSUE THREAD

**Issue by zxlin** [fomantic/Fomantic-UI#1908]
# Bug Report

A header with icon class produces an unnecessary padding-left (looks to be left over from the case where icon and text are on the same line)

This padding-left is only necessary on a `header` without the `icon` class.

Possible offending lines: https://github.com/fomantic/Fomantic-UI/blob/fcd221b001d45ec6bc820d5f43a3d8bcad377bf0/dist/components/header.css#L101-L106

## Steps to reproduce
```HTML
<h2 class="ui icon header">
  <i class="settings icon"></i>
  <div class="content">
    Account Settings
    <div class="sub header">Manage your account settings and set e-mail preferences.</div>
  </div>
</h2>
```

## Expected result
Should not have `padding-left` on the `content` div. 

## Actual result
`padding-left` exists, creates off centered header.

## Testcase

Bug present in the docs itself: https://fomantic-ui.com/elements/header.html#icon-headers

## Screenshot (if possible)
<img width="1009" alt="Screen Shot 2021-03-05 at 2 34 48 PM" src="https://user-images.githubusercontent.com/5447705/110164852-f22a0e00-7dbf-11eb-989d-14605031e177.png">

## Version

2.8.7

**Comment by lubber-de** [comment]
Fixed by #1909

**Comment by zxlin** [comment]
Thanks for addressing this so quickly!
